docs/DEVELOPING.md

```diff
@@ -441,6 +441,106 @@ $ make ftest NAME=sharepoint
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+All services and records the user has access to will be indexed according to the configurations provided.
+
+### Usage
+
+To use this connector as a **connector client**, use the **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+### Compatibility
+
+ServiceNow "Tokyo", "San Diego" & "Rome" versions are compatible with Elastic connector frameworks.
+
+### Configuration
+
+The following configuration fields are required to set up the connector:
+
+##### `url`
+
+The host url of the ServiceNow instance.
+
+##### `username`
+
+The username of the account for ServiceNow.
+
+##### `password`
+
+The password of the account used for ServiceNow.
+
+##### `services`
+
+Comma-separated list of services to fetch data from ServiceNow. If the value is `*`, the connector will fetch data from the list of basic services provided by ServiceNow: 
+- [User](https://docs.servicenow.com/bundle/utah-platform-administration/page/administer/roles/concept/user.html)
+- [Incident](https://docs.servicenow.com/bundle/tokyo-it-service-management/page/product/incident-management/concept/c_IncidentManagement.html)
+- [Requested Item](https://docs.servicenow.com/bundle/tokyo-servicenow-platform/page/use/service-catalog-requests/task/t_AddNewRequestItems.html)
+- [Knowledge](https://docs.servicenow.com/bundle/tokyo-customer-service-management/page/product/customer-service-management/task/t_SearchTheKnowledgeBase.html)
+- [Change Request](https://docs.servicenow.com/bundle/tokyo-it-service-management/page/product/change-management/task/t_CreateAChange.html)
+
+Default value is `*`. Examples:
+
+  - `User, Incident, Requested Item, Knowledge, Change Request`
+  - `*`
+
+##### `retry_count`
+
+The number of retry attempts after a failed request to ServiceNow. Default value is `3`.
+
+##### `concurrent_downloads`
+
+The number of concurrent downloads for fetching the attachment content. This speeds up the content extraction of attachments. Defaults to `10`.
+
+### Documents and syncs
+
+The connector syncs the following ServiceNow object types: 
+- **Records**
+- **Attachments**
+
+### Sync rules
+
+- Content of files bigger than 10 MB won't be extracted.
+- Permissions are not synced. **All documents** indexed to an Elastic deployment will be visible to **all users with access** to that Elastic Deployment.
+- Advanced sync rules are not available in the present version. Currently filtering is controlled via ingest pipelines.
```

Comment:
Sync rules should be part of the 8.9 ServiceNow implementation.

Reply:
Yes, Once we raise Sync rules PR will updating documentation accordingly.